Clarify what should be sent on PTO (#3897)
quicwg/base-drafts: draft-ietf-quic-recovery.md

If the sender wants to elicit a faster acknowledgement on PTO, it can skip a
packet number to eliminate the ack delay.

> When the PTO timer expires, new or previously sent unacknowledged data MUST
> be sent if available. A probe packet SHOULD carry new data when possible.
> A probe packet MAY carry retransmitted unacknowledged data when new data is
> unavailable, when flow control does not permit new data to be sent, or to
> opportunistically reduce loss recovery delay.  Implementations MAY use

Reviewer: ```suggestion
When the PTO timer expires, an ack-eliciting packet MUST be sent.  An endpoint
SHOULD include new data in this packet.  Previously sent data MAY be sent if
no new data can be sent.  Implementations MAY use
```

Reviewer 2: Thanks. I think this is more concise. Additionally,

"When the PTO timer ..." -> L625 says "When a PTO timer ..."
"an ack-eliciting packet MUST be sent" -> is it a duplicate of L625?
"An endpoint SHOULD include new data in this packet" -> "An endpoint SHOULD include new data in this packet if available"

Reviewer 2: Also, L649 (below this line, I can't comment on the line) says "sending new or retransmitted" -> "sending new or previously-sent"

Reviewer: Yeah the duplication with earlier is bad.

```suggestion
An endpoint SHOULD include new data in probe packets.  Data that was sent
previously MAY be sent again no new data can be sent.  Implementations MAY use
```

I think that using previously-sent and retransmitted is OK.
